doc/content/tutorials/convergence.md

```diff
@@ -62,14 +62,15 @@ To determine an appropriate number of inactive batches, you can use the
   of cell divisions to create
 - A Cardinal input file that runs your OpenMC model given some imposed
   temperature and density distribution
+- Criteria used to assess the convergence of the fission source
```

Comment:
I would remove this - it's not really a "need" in the same way that an input file is a hard, tangible need.